dask/array/tests/test_sparse.py

```diff
@@ -168,6 +168,8 @@ def test_metadata():
         assert isinstance(np.concatenate([y, y])._meta, sparse.COO)
         assert isinstance(np.concatenate([y, y[:0], y])._meta, sparse.COO)
         assert isinstance(np.stack([y, y])._meta, sparse.COO)
+        if np.__version__ >= "1.17":
```

Comment:
Can I bother you to make a `_numpy_117` variable in `dask/array/numpy_compat.py` and use that here? I find it easier to clean things up down the road when all the versions-specific things refer to variables.

Reply:
Sounds reasonable Tom. Have pushed such a change. Please check and see if this addresses your concern.